app/views/projects/dataset/keys/_numeric_key.html.erb

```diff
@@ -1,4 +1,4 @@
 <div class="dataset-key">
   <%= f.label :value, key.title %>
-  <%= f.text_field :value, rows: 5 %>
```

Comment:
Not a great error message on this:

![Screenshot 2020-07-09 at 17 54 44](https://user-images.githubusercontent.com/282788/87068337-508cf580-c20d-11ea-902f-06b5eb375999.png)

Is there a reason not to use [`number_field`](https://api.rubyonrails.org/v5.1.7/classes/ActionView/Helpers/FormTagHelper.html#method-i-number_field_tag)?

![Screenshot 2020-07-09 at 17 56 55](https://user-images.githubusercontent.com/282788/87068560-a6619d80-c20d-11ea-8eb6-27647d5d6e46.png)